Tests/Mocks/API/api.zeplin.dev_v1_styleguides_5e221416f680ef6a5ce43bdf_text_styles.json

```diff
@@ -9,6 +9,7 @@
         "font_weight": 700,
         "font_style": "normal",
         "font_stretch": 1,
+        "letter_spacing": 0.408,
```

Comment:
mind changing this to 0.401? I basically wanna make sure it rounds down to `0.4`.

Reply:
Sure!